interface/ceed-qfunctioncontext.c

```diff
@@ -74,6 +74,19 @@ int CeedQFunctionContextRegisterGeneric(CeedQFunctionContext ctx, const char *fi
   }
   CeedCall(CeedCalloc(1, &ctx->field_labels[ctx->num_fields]));
 
+  // Compute field size
+  switch (field_type) {
+    case CEED_CONTEXT_FIELD_DOUBLE:
+      field_size = sizeof(double);
+      break;
+    case CEED_CONTEXT_FIELD_INT32:
+      field_size = sizeof(int);
+      break;
```

Comment:
Seems odd to me to specify int32 in the enum when we're technically relying on `int` to be 32 bit. Granted, 99% of the time that is true, but maybe we could explicitly specify `int32_t` or just use `CeedInt`?

Side note, any reason why the context fields don't use `CeedScalar`/`CeedInt` instead of compiler-generic values? I think most of the time we assume that `double` == `CeedScalar` and `int` == `CeedInt` in the QF context, but that might not always be true I think.

Reply:
I use `int` and `double` because I want to be explicit about how many bytes are being copied. A `PetscScalar` and `CeedScalar` or a `PetscInt` and a `CeedInt` can be completely different types.